provision/acc_provision/acc_provision.py

```diff
@@ -57,6 +57,31 @@
     },
 }
 
+# Known Flavor options:
+# - template_generator: Function that generates the output config
+#       file. Default: generate_kube_yaml.
+# - version_fields: List of config options that must be specified for
+#       the specific version of deployment. Default: VERSION_FIELDS.
+# - apic: Dict that is used for configuring ApicKubeConfig
+#       Known sub-options:
+#       - use_kubeapi_vlan: Whether kubeapi_vlan should be used. Default: True.
+#       - tenant_generator: Name of the function to generate tenant objects.
+#             Default: kube_tn.
+#       - associate_aep_to_nested_inside_domain: Whether AEP should be attached
+#             to nested_inside domain. Default: False.
+KubeFlavorOptions = {}
+
+CfFlavorOptions = {
```

Comment:
I'm completely unable to see why this is supposed to be better than just putting these with all the other options?